docs/pt/docs/tutorial/request-forms.md

```diff
@@ -0,0 +1,58 @@
+# Dados do formulário
+
+Quando você precisa receber campos de formulário em vez de JSON, você pode usar `Form`.
+
+!!! info "Informação"
+    Para usar formulários, primeiro instale <a href="https://andrew-d.github.io/python-multipart/" class="external-link" target="_blank">`python-multipart`</a>.
+
+    E.g. `pip install python-multipart`.
+
+## Importe `Form`
+
+Importe `Form` de `fastapi`:
+
+```Python hl_lines="1"
+{!../../../docs_src/request_forms/tutorial001.py!}
+```
+
+## Declare parâmetros de `Form`
+
+Crie parâmetros de formulário da mesma forma que você faria para `Body` ou `Query`:
+
+```Python hl_lines="7"
+{!../../../docs_src/request_forms/tutorial001.py!}
+```
+
+Por exemplo, em uma das maneiras que a especificação OAuth2 pode ser usada (chamada "fluxo de senha"), é necessário enviar um `username` e uma `password` como campos de formulário.
```

Comment:
"como campos do formulário"
we are referring to a specific form passed in the request

Reply:
Done.